src/server.ts

```diff
@@ -1,38 +1,11 @@
-import Koa from "koa";
-import bodyParser from "koa-bodyparser";
-import cors from "koa2-cors";
-import logger from "koa-logger";
-import Router from "koa-router";
+require('dotenv').config()
+import app from "./app/app"
+import connectToDB from "./database/database.connection"
```

Comment:
You export not `connectToDB` function, but result of its call. Also it is a promise, so the check on line 7 is always true. Basically it is very easy to check if what you did for connection is correct. Just pass incorrect credentials for DB and see if you app starts or log the error as expected.